client/js/views/visualize.react.jsx

```diff
@@ -83,7 +78,7 @@ var Visualize = React.createClass({
   },
 
   getInitialState: function() {
-    return {commits: [], commitIndex: 0, currentPath: '', fileTree: {}, filePaths : {}};
+    return {windowHeight: $(window).height() - 285, commits: [], commitIndex: 0, currentPath: '', fileTree: {}, filePaths : {}};
```

Comment:
window height needs to be state. window.onresize should change the state and re-render the frame size